lib/galaxy/datatypes/converters/bigbed_to_gtf_converter.xml

```diff
@@ -0,0 +1,123 @@
+<tool id="bigbedtogtf" name="Convert from bigBed to GTF format" version="@TOOL_VERSION@+galaxy@SUFFIX_VERSION@" profile="20.01">
+    <description>Convert bigBed to GTF</description>
+    <macros>
+        <token name="@TOOL_VERSION@">377</token>
+        <token name="@SUFFIX_VERSION@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">ucsc-bigbedtobed</requirement>
+        <requirement type="package" version="3.7">python</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ## Step 1: Convert BigBed to BED
+        bigBedToBed '${bigbed_input}'
+        #if $chrom
+            -chrom='${chrom}'
+        #end if
+        #if str($start)
+            -start=$start
+        #end if
+        #if str($end)
+            -end=$end
+        #end if
+        temp.bed &&
+
+        ## Step 2: Convert BED to GFF using the existing converter script
+        python '$__tool_directory__/bed_to_gff_converter.py' temp.bed temp.gff &&
```

Comment:
Is there a reason you're not using `genePredToGtf` ? I don't know if we can trust that 17 year old hand-written bed to gff script ...

Reply:
i was partly trying to be consistent w what was already here. it seemed to me like itd be unexpected as a user if i got a very different gtf vs gff2 file w the same input. but also, i was a little worried that if the incoming file wasnt from ucsc that the genepred intermediate file wouldnt make sense or maybe even work. (im not really confident i know what genepred is tbh) and if we did know the incoming data was from ucsc, why not have a genepred to gtf converter?

Reply:
Agree these are issues, but realistically I think all bigbeds come from ucsc. That said, I think the bigbed to bed converter was the important one that would have an application, maybe we can wait until there's a reason not to use the pre-built gtfs hosted by ucsc before we proceed here ? If there is an actual application we'll also know if this will all work out correctly.